packages/plugin-ext/src/main/browser/status-bar-message-registry-main.ts

```diff
@@ -64,7 +68,7 @@ export class StatusBarMessageRegistryMainImpl implements StatusBarMessageRegistr
             color: color && (this.colorRegistry.getCurrentColor(color) || color),
             // In contrast to color, the backgroundColor must be a theme color. Thus, do not hand in the plain string if it cannot be resolved.
             backgroundColor: backgroundColor && (this.colorRegistry.getCurrentColor(backgroundColor)),
-            tooltip,
+            tooltip: tooltip === true ? (token: CancellationToken) => this.proxy.$getMessage(id, token) : tooltip,
```

Comment:
nit: Maybe add a comment here, as `tooltip === true` is not obvious on the first glance to everyone.